tools/openapi.yaml

```diff
@@ -15,6 +15,88 @@ info:
 servers:
 - url: https://vcell-dev.cam.uchc.edu
 paths:
+  /api/v1/Simulation/{simID}/simulationStatus:
```

Comment:
add query parameter(s) for mathmodel or biomodel refererence to provide permission for shared BioModels and MathModels (access by reachability)

Reply:
Made shared models simulation status accessible.